Classes/VTAcknowledgementsViewController.m

```diff
@@ -232,7 +234,7 @@ - (void)configureHeaderView
     UILabel *label = [[UILabel alloc] initWithFrame:labelFrame];
     label.text             = self.headerText;
     label.font             = font;
-    label.textColor        = [UIColor grayColor];
+	label.textColor        = self.textColor ?: [UIColor grayColor];
```

Comment:
@moritzmoritz Thanks... will convert tabs to spaces, as you like. I don't understand your other comment about the `?`, however. Can you clarify please?